docs/preview/features/powershell/arm.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Scripts related to ARM templates"
+layout: default
+---
+
+# ARM
+
+## Installation
+
+To have access to the following features, you have to import the module:
+
+```powershell
+PS> Import-Module -Name Arcus.Scripting.ARM
+```
+
+## Inject content in ARM template
+
+In certain scenarios, you have to embed content into an ARM template to deploy it.
+However, the downside of it is that it's buried inside the template and tooling around it might be less ideal - An example of this is OpenAPI specifications you'd want to deploy.
+By using this script, you can inject external files inside your ARM template.
+
+| Parameter | Mandatory | Description                                                         |
+| --------- | --------- | ------------------------------------------------------------------- |
+| `Path`    | no        | The file path to the ARM template to inject the external files into |
```

Comment:
I would say it's mandatory since it can't run otherwise, no?

Reply:
Ah yes I have to add this. Apparently they use `$PSScriptPath` as default value.

Reply:
Ah ok, that's ok but should be documented then indeed!

Reply:
Yes, done.